packages/fxa-settings/src/models/Account.ts

```diff
@@ -6,11 +6,11 @@ import {
   QueryLazyOptions,
 } from '@apollo/client';
 
-export interface Location {
-  city: string;
-  country: string;
-  state: string;
-  stateCode: string;
+export interface DeviceLocation {
+  city?: string;
+  country?: string;
+  state?: string;
+  stateCode?: string;
```

Comment:
these should technically be like `city: string | null;` we discovered `undefined` doesn't work